client/platform/web-girder/constants.ts

```diff
@@ -6,7 +6,7 @@ import { DatasetMeta, DatasetMetaMutable, DatasetType } from 'dive-common/apispe
  */
 interface GirderMetadataStatic extends DatasetMetaMutable {
   id: string;
-  type: Readonly<DatasetType>;
+  type: Readonly<DatasetType | 'multi'>;
```

Comment:
Need to clarify where this should be integrated into DatasetType or if it should remain separate there are arguments for both?